mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1065,6 +1084,7 @@ pub async fn nexus_create(
     size: u64,
     uuid: Option<&str>,
     children: &[String],
+    nvme_params: Option<(u16, u16, u64)>,
```

Comment:
This is rather cumbersome no? If we put this in a struct it is much easier later one when we add/remove arguments?